homeassistant/components/number/const.py

```diff
@@ -342,6 +344,15 @@ class NumberDeviceClass(StrEnum):
     """
 
 
+class NumberMode(StrEnum):
+    """Modes for number entities."""
+
+    AUTO = "auto"
+    BOX = "box"
+    SLIDER = "slider"
+
+
+DEVICE_CLASSES_SCHEMA: Final = vol.All(vol.Lower, vol.Coerce(NumberDeviceClass))
```

Comment:
Is that needed?
If yes, moving to const is a breaking change. 
If not, remove it?